Source/ChilliSource/Core/Base/Application.cpp

```diff
@@ -508,7 +556,7 @@ namespace ChilliSource
     //------------------------------------------------------------------------------
     void Application::Render() noexcept
     {
-        m_renderer->ProcessRenderCommandBuffer();
+        m_renderer->ProcessRenderCommandBuffers();
```

Comment:
Sorry that was a hangover from an early change. I was sure that I had reverted that